slsim/Sources/source.py

```diff
@@ -90,3 +90,19 @@ def magnitude(self, band, image_observation_times=None):
         else:
             source_mag = self.source_dict[band_string]
             return source_mag
+
+    def es_magnitude(self, band):
```

Comment:
I would rename the definition to something better interpretable. 'es' is hard to understand. You can also write it out as extended_source_magnitude()